eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -44,6 +44,11 @@ Param (
 
 . (Join-Path $PSScriptRoot common.ps1)
 
+Write-Host "Service directory: $serviceDirectory"
```

Comment:
Clean these up before merge? I assume they're present to see how the branches run